design/images/container.puml

```diff
@@ -0,0 +1,29 @@
+@startuml
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
+
+Person(user_request_input, "Data input and request")
+Person(user_admin, "Admin")
+Person_Ext(user_public, "Public (unauthorized)")
+
+System_Boundary(seedcase, "Seedcase instance for Data Resource") {
+  System(web_framework, "Web Interface")
+  System(id_management, "User ID Authentication")
+  System(api_framework, "API Layer")
+  System(backend_database, "Database")
+  System(backend_file_storage, "File Storage")
+  System(backend_large_file_storage, "Large File Storage")
+
+}
+System_Ext(external_server, "External secure server")
+
+Rel(user_public, web_framework, "View general details about Data Resource")
+Rel(user_request_input, web_framework, "Input to or request data from Data Resource")
+Rel(user_admin, web_framework, "Create, manage, and update data projects")
+Rel(id_management, web_framework, "User Authetication")
+Rel(web_framework, id_management, "")
+Rel(web_framework, api_framework, "Send user requests using API communication protocols")
+Rel(api_framework, backend_database, "Database-specific commands")
+Rel(api_framework, backend_file_storage, "Request to get, create, update, or remove files")
+Rel(api_framework, backend_large_file_storage, "Request to get, create, update, or remove files")
+Rel(api_framework, external_server, "Data extract for approved data projects")
+@enduml
```

Comment:
I've suggested some descriptions to the various boxes. 

@startuml

 !include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml

 Person(user_request_input, "Data user", "Researcher, Data Manager")
 Person(user_admin, "Admin", "Superuser and/or Data Controller")
 Person_Ext(user_public, "Public (unauthorized)", "Interested third party")

 System_Boundary(seedcase, "Seedcase instance for Data Resource") {
   System(web_framework, "Web Interface")
   System(id_management, "User ID Authentication")
   System(api_framework, "API Layer")
   System(backend_database, "Database", "Project data, Meta data, User permissions")
   System(backend_file_storage, "File Storage", "Archive of data files uploaded into db")
   System(backend_large_file_storage, "Large File Storage", "Parquet files, image files")

 }
 System_Ext(external_server, "External secure server", "eg. university or hospital servers")

 Rel(user_public, web_framework, "View general details about Data Resource")
 Rel(user_request_input, web_framework, "Input to or request data from Data Resource")
 Rel(user_admin, web_framework, "Create, manage, and update data projects")
 Rel(id_management, web_framework, "User Authetication")
 Rel(web_framework, id_management, "")
 Rel(web_framework, api_framework, "Send user requests using API communication protocols")
 Rel(api_framework, backend_database, "Database-specific commands")
 Rel(api_framework, backend_file_storage, "Request to get, create, update, or remove files")
 Rel(api_framework, backend_large_file_storage, "Request to get, create, update, or remove files")
 Rel(api_framework, external_server, "Data extract for approved data projects")
@enduml